source/main.cpp

```diff
@@ -55,11 +56,34 @@ std::string FormatDeviceCaps(vk::PhysicalDevice PhysicalDevice);
 
 int main(int argc, char* argv[])
 {
+	std::ifstream ifs("settings.json");
+    Json::Reader reader;
+    Json::Value obj;
+    reader.parse(ifs, obj);
```

Comment:
These variable names do not match the [PascalCase](https://techterms.com/definition/pascalcase) naming pattern found in the rest of the code base.

Reply:
Fixed in [bbc8649](https://github.com/Wunkolo/vkblam/pull/1/commits/bbc8649cf4c93550cbd1a16611d7e3a3c108c38b). Closing.